### PR TITLE
CB-8227 CB-8237 CB-8238 Add --save option to 'cordova platform add', 'cordova platform remove' and 'cordova platform update'

### PR DESCRIPTION
**Pull request by omefire**
CB-8227 CB-8237 CB-8238 Add --save option to 'cordova platform add', 
    'cordova platform remove' and 'cordova platform update'

### CODE REVIEW

**Comment by omefire** [comment]
@gorkem , plz review

File: src/cli.js [hunk 1]
```diff
@@ -286,6 +286,7 @@ function cli(inputArgs) {
                             , cli_variables : cli_vars
                             , browserify: args.browserify || false
                             , link: args.link || false
+			    , save: args.save || false
```
Comment (jsoref) [inline src/cli.js]: offhand, this looks unaligned

**Comment by omefire** [comment]
@jsoref , thanks! space issue fixed.

**Comment by omefire** [comment]
related PR: https://github.com/apache/cordova-lib/pull/144

**Pull request merged**